keras/utils/data_utils.py

```diff
@@ -186,19 +186,18 @@ def get_file(fname,
 
     if download:
         print('Downloading data from', origin)
-        progbar = None
+        enclosed = {'progbar': None}
```

Comment:
Don't know the keras policy on this, but would comment with a link referencing an explanation of this workaround be appropriate? Otherwise someone might submit a pull request to remove the "unnecessary" dictionary.